mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -440,6 +440,10 @@
     <string name="tabs_header_private_tabs_title">Private tabs</string>
     <!-- Content description (not visible, for screen readers etc.): Add tab button. Adds a news tab when pressed -->
     <string name="add_tab">Add tab</string>
+    <!-- Content description (not visible, for screen readers etc.): Add tab button. Adds a news tab when pressed -->
+    <string name="add_private_tab">Add private tab</string>
+    <!-- Content for FAB when looking at private tabs in the tab drawer -->
```

Comment:
@boek 
What is FAB? Can we update the comment to replace it with a description?